lxc/list.go

```diff
@@ -131,6 +133,7 @@ lxc list -c ns,user.comment:comment
 	cmd.Flags().StringVarP(&c.flagColumns, "columns", "c", defaultColumns, i18n.G("Columns")+"``")
 	cmd.Flags().StringVarP(&c.flagFormat, "format", "f", "table", i18n.G("Format (csv|json|table|yaml)")+"``")
 	cmd.Flags().BoolVar(&c.flagFast, "fast", false, i18n.G("Fast mode (same as --columns=nsacPt)"))
+	cmd.Flags().BoolVar(&c.flagAllProjects, "all-projects", false, i18n.G("Display isntances from all projects"))
```

Comment:
We should alter the default set of column so that if `--all-projects` is passed and the user doesn't specify a specific set of columns (though either `-c` or `--fast`), then we should include the project column by default as first column.

Reply:
`Project` column added when `--all-project` flag specified.